Content.Shared/Armable/ArmableSystem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Examine;
+using Content.Shared.Toggleable;
+using Content.Shared.Item.ItemToggle.Components;
+
+namespace Content.Shared.Armable;
+
+public sealed class ArmableSystem : EntitySystem
+{
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ArmableComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<ArmableComponent, ItemToggleActivateAttemptEvent>(TryArming);
+        SubscribeLocalEvent<ArmableComponent, ItemToggledEvent>(ArmingDone);
+    }
+
+    private void TryArming(Entity<ArmableComponent> entity, ref ItemToggleActivateAttemptEvent args)
+    {
+        if (TryComp<ItemToggleComponent>(entity, out var comp))
+            comp.Activated = true;
+    }
+
+    private void OnExamine(EntityUid uid, ArmableComponent comp, ExaminedEvent args)
+    {
+        if (!args.IsInDetailsRange || !comp.ShowStatusOnExamination || !TryComp<ItemToggleComponent>(uid, out var itemToggle))
+            return;
+
+        if(itemToggle.Activated)
+            args.PushMarkup(Loc.GetString("examine-armed", ("name", uid)));
+        else
+            args.PushMarkup(Loc.GetString("examine-not-armed", ("name", uid)));
```

Comment:
Could you move these LocIds into `ArmableComponent`?

Example:

https://github.com/space-wizards/space-station-14/blob/2ff82380301f8ea61f021ecebc8d5dc6daebdeb6/Content.Shared/Wires/WiresPanelComponent.cs#L45-L57

Reply:
Fixed